front/src/App.tsx

```diff
@@ -1,56 +1,74 @@
 import { Navigate, Route, Routes } from 'react-router-dom';
 
+import { SettingsLayout } from '@/ui/layout/SettingsLayout';
+
 import { RequireAuth } from './modules/auth/components/RequireAuth';
 import { AppLayout } from './modules/ui/layout/AppLayout';
 import { AuthCallback } from './pages/auth/AuthCallback';
 import { Login } from './pages/auth/Login';
 import { Companies } from './pages/companies/Companies';
 import { Opportunities } from './pages/opportunities/Opportunities';
 import { People } from './pages/people/People';
+import { SettingsProfile } from './pages/settings/SettingsProfile';
+import { SettingsWorkspace } from './pages/settings/SettingsWorkspace';
 
 export function App() {
   return (
-    <>
-      {
-        <AppLayout>
-          <Routes>
-            <Route
-              path="/"
-              element={
-                <RequireAuth>
-                  <Navigate to="/people" replace />
-                </RequireAuth>
-              }
-            />
-            <Route
-              path="/people"
-              element={
-                <RequireAuth>
-                  <People />
-                </RequireAuth>
-              }
-            />
-            <Route
-              path="/companies"
-              element={
-                <RequireAuth>
-                  <Companies />
-                </RequireAuth>
-              }
-            />
-            <Route
-              path="/opportunities"
-              element={
-                <RequireAuth>
-                  <Opportunities />
-                </RequireAuth>
-              }
-            />
-            <Route path="/auth/callback" element={<AuthCallback />} />
-            <Route path="/auth/login" element={<Login />} />
-          </Routes>
-        </AppLayout>
-      }
-    </>
+    <Routes>
+      <Route
+        path="/*"
+        element={
+          <AppLayout>
```

Comment:
I wonder if AppLayout should be called from here or from within pages

Reply:
I feel it's wrong that we are having to set of Routes because of Layout.

Can we do:
```
<Routes>
<Route>
<RequireAuth>
<People> (resp <Settings>)
<AppLayout> (resp <SubNavLayout>)
```

Reply:
I think that's what I tried at first and it didn't work. I'll give it another shot and let you know

Reply:
Yep, we anticipated that we might have to rework AppLayout structure in this case. Not sure to understand all the impacts of my comments :p